lambdas/ecs-events-to-slack/notify.py

```diff
@@ -22,10 +22,17 @@ def message_formatter(event, region):
     attachment_status = event_detail["attachments"][0]["status"]
 
     env_vars = event_detail["overrides"]["containerOverrides"][0]["environment"]
+    excludes = {}
+    if "ENV_EXCLUDES" in os.environ:
+        excludes = os.environ["ENV_EXCLUDES"].replace(" ", "").split(",")
     env_vars_list = []
     for env_var in env_vars:
-        env_var_formatted = "%s=\"%s\""
-        % (env_var["name"], env_var["value"])
+        if env_var["name"] in excludes:
+            env_var_formatted = "%s=\"%s\""
+            % (env_var["name"], "REDACTED")
```

Comment:
Can we write this all on one line, splitting the line at `%` looks very unutual in python.

Reply:
I also think you shouldn't put quotes around the value for this case, so it can simplify to `env_var_formatted = "%s=REDACTED" % env_var["name"]`

Reply:
I was going with the style already in the file, but I prefer it on one line as well.... so I just updated.